packages/engine-api-client/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@lodestar/engine-api-client",
+  "version": "1.2.1",
+  "description": "Client implementation of the execution layer's Engine API",
+  "author": "ChainSafe Systems",
+  "type": "module",
+  "homepage": "https://github.com/ChainSafe/lodestar#readme",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ChainSafe/lodestar.git"
+  },
+  "exports": "./lib/index.js",
+  "types": "lib/index.d.ts",
+  "typesVersions": {
+    "*": {
+      "*": [
+        "*",
+        "lib/*",
+        "lib/*/index"
+      ]
+    }
+  },
+  "files": [
+    "lib/**/*.d.ts",
+    "lib/**/*.js",
+    "lib/**/*.js.map",
+    "*.d.ts",
+    "*.js"
+  ],
+  "scripts": {
+    "clean": "rm -rf lib && rm -f *.tsbuildinfo",
+    "build": "tsc -p tsconfig.build.json",
+    "build:lib:watch": "yarn run build:lib --watch",
+    "build:release": "yarn clean && yarn build",
+    "build:types:watch": "yarn run build:types --watch",
+    "check-build": "node -e \"(async function() { await import('./lib/index.js') })()\"",
+    "check-types": "tsc",
+    "lint": "eslint --color --ext .ts src/ test/",
+    "lint:fix": "yarn run lint --fix",
+    "test:unit": "mocha 'test/unit/**/*.test.ts'",
+    "check-readme": "typescript-docs-verifier"
+  },
+  "bugs": {
+    "url": "https://github.com/ChainSafe/lodestar/issues"
+  },
+  "dependencies": {
+    "@lodestar/utils": "^1.2.1",
+    "@lodestar/types": "^1.2.1",
+    "@lodestar/config": "^1.2.1",
+    "@lodestar/db": "^1.2.1",
+    "@lodestar/state-transition": "^1.2.1",
+    "cross-fetch": "^3.1.4",
+    "jwt-simple": "0.5.6",
+    "c-kzg": "^1.0.7"
```

Comment:
- This will break Lodestar installations, see https://github.com/ChainSafe/lodestar/issues/4886 c-kzg must not be installed (for now) by default.
- It was fixed in https://github.com/ChainSafe/lodestar/pull/4888 but this commits undo the fix